docs/1.0. Overview.md

```diff
@@ -14,4 +14,4 @@ A media consuming unit associated with an IS-04 Receiver. Sink is a resource and
 
 ### Media Profile
 
-Media description which describes a format acceptable for all Receivers implied to be a part of a connection. A Media Profile consist of separate parameters required to set up Sender and such IS-04 resources behind it as Source and Flow.
+A Collection of formats that are acceptable for all Receivers implied to be a part of a connection. A Media Profile consist of separate parameters required to set a Sender, Flow, and/or Source.
```

Comment:
```suggestion
Description of a format that is acceptable for all Receivers implied to be a part of a connection. A Media Profile consist of separate media parameters required to set a Sender, Flow, and/or Source.
```
It is a single format, not a collection.

Reply:
:confused: https://github.com/AMWA-TV/nmos-sink-metadata-processing/blob/75e292aa65a28d63a057a0fded694cb0d30a47f8/APIs/schemas/media_profiles.json#L3